src/composers/field.tsx

```diff
@@ -59,10 +62,11 @@ export function Field<Props>(WrappedComponent: IncomingField<Props>,
         }
 
         public render() {
+            const fieldState = this.props.form.getFieldState(this.getFieldName());
```

Comment:
So we would push the local state to the bonn form, so instead of setting the local state in the `keyPressHandler` we push it to bonn and let it forceUpdate any listening components? Sounds a bit tricky tho. Gotta be carefull to not render too much (e.g. having mutliple things inside a field would already force multiple renders). But I guess that would also mean a user mis-used the field decorator.